.travis.yml

```
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 10.16.3
```

I don't think it make sense to worry about anything other then semver-major version here

```suggestion
  - 10
```